src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.search-bar {
+  position: relative;
+  margin-top: 20px;
+  align-items: center;
+}
+
+.search-bar__input {
+  display: flex;
```

Comment:
The 'display: flex' property is not really applicable for an input element. Remove this property if it's not necessary.